home/templatetags/home_tags.py

```diff
@@ -143,3 +143,17 @@ def render_redirect_from_with_help_text(field):
                       f'See "https://docs.wagtail.io/en/stable/editor_manual/managing_redirects.html" for more details.'
 
     return {'field': field, 'red_help_text': True}
+
+
+@register.inclusion_tag('home/tags/image.html', takes_context=True)
+def render_image(context, image, half_width=False, img_class=None):
+    width = SiteSettings.get_for_default_site().maximum_width
```

Comment:
Shouldn't this be `default_image_resize_rule_width`?

Reply:
This needs to be absolutely bullet-proof. If anything goes wrong (no site settings, no default site) we should fall back to a sensible default setting.